proposed/2021/NuGetConsumePDBInPackageReference.md

```diff
@@ -0,0 +1,216 @@
+# Consuming pdb support with PackageReference
+
+* Status: Reviewing
+* Author(s): [Heng Liu](https://github.com/heng-liu)
+* Issue: [5926](https://github.com/NuGet/Home/issues/5926) Consume pdbs from packages in PackageReference - add Debug Symbols to LockFileTargetLibrary
+* Type: Feature
+
+## Problem Background
+
+In the PackageReference world, the packages are consumed from the global-packages folder, .pdb, .xml files from lib and runtime folder are not copied into the build output folder.
+In some cases, consuming .pbd, .xml files are needed to support certain features, like source link ( see issue [1458](https://github.com/dotnet/sdk/issues/1458)).
```

Comment:
```suggestion
In some cases, consuming .pdb, .xml files are needed to support certain features, like debugging ( see issue [1458](https://github.com/dotnet/sdk/issues/1458)).
```

I realize I said `source link` in one of my comments last time, which is not true.
mb.

Reply:
Thanks! Fixed.